### PR TITLE
{geo}[gompi/2021b] GLM-AED v3.3.0a5

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GLM-AED/GLM-AED-3.3.0a5-gompi-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+stratification hydrodynamic model, which is dynamically coupled with the AED
+water quality modelling library. GLM-AED is suitable for simulating conditions
+in a wide range of natural and engineered lakes, including shallow (well-mixed)
+and deep (stratified) systems. The model has been successfully applied to
+systems from the scale of individual ponds and wetlands, to actively operated
+reservoirs, upto the scale of the Great Lakes."""
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+
+source_urls = ['https://github.com/AquaticEcoDynamics/glm-aed/archive/']
```
Comment (jfgrimm) [inline easybuild/easyconfigs/g/GLM-AED/GLM-AED-3.3.0a5-gompi-2021b.eb:17]:
```suggestion
```

Comment (lexming) [reply easybuild/easyconfigs/g/GLM-AED/GLM-AED-3.3.0a5-gompi-2021b.eb]: thanks for the fix!

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.4
See https://gist.github.com/a30ad1834832f1bbb1982be96c3ec720 for a full test report.

**Comment by bedroge** [comment]
```
no change to version number in ./glm.rc
no change to version number in ./glm+.rc
debian version 3.3.0a5
dh_testdir
make: dh_testdir: No such file or directory
make: *** [debian/rules:11: binary] Error 127
```

Looks like it's trying to build a deb package on Debian-based systems:
https://github.com/AquaticEcoDynamics/glm-aed/blob/main/glm-source/build_glm.sh#L245
Maybe we should just remove that part?

**Comment by lexming** [comment]
@bedroge I think I fixed it, please test it again.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.4
See https://gist.github.com/9456d5a3458cd9be15a7c1a6ebceb826 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/b8dbb9627d336831fd96561f475ceb9e for a full test report.

**Comment by bedroge** [comment]
Going in, thanks @lexming!

**Pull request merged**